### ISSUE TITLE
OpenModelica 1.19.2 installation on windows does no longer install MSL

### ISSUE THREAD

**Issue by dietmarw** [OpenModelica/OpenModelica#9369]
### Description
I was just testing the windows install of OpenModelica in order to update some installation instructions for a workshop. I noticed that the "choose your MSL version" dialogue is gone and after a successful installation you end up having a "naked" OMEdit without any libraries installed. Not even MSL. 

I now need to instruct my students to first install MSL via the package manager **and** also go into the options in order to set it up to get loaded default. This is a massive step-back for the user experience.

### Steps to Reproduce

Fresh install of OpenModelica (with previous settings deleted)

### Expected Behavior

The installer should:
 1. either show the MSL choice dialogue or install at least MSL 4.0.0 by default
 2. should set-up OMedit such that *a* MSL is always loaded upon startup.

### Version and OS
1.19.2 on Windows 10

**Comment by casella** [comment]
@dietmarw, thanks for the feedback!

In fact, the fresh install on Windows has a default _installed_ version of *both* MSL 3.2.3 and MLS 4.0.0, in the installation directory but it's not _loaded_ by default. You can load them from the Welcome tab if you click on the System Libraries button, sorry if this was not immediately clear from the interface design. On the other hand, the old splash screen was really ugly, you shouldn't really need to go through all this text to use a GUI 😅.

Anyway, we are fully aware that the handling of MSL in 1.19.x is still messy, mainly because we are mixing up preinstalled versions of MSL with the ones you get from the package manager, so that later you see two versions of each, which is a bit confusing. Also, there is no longer any reason not to load one version of MSL by default, because now, if you then load a library that uses an incompatible one, you get a dialog to automatically unload the loaded version of MSL and load the right one, with just one click. This was not the case with previous versions of OMEdit, where the wrong version of MSL was kept, you got a bogus error message, and nothing worked at all. This issue has quite some ramifications, so getting it right is not trivial and we couldn't make it in 1.19.x.

We planned to fix this situation once and for all in the forthcoming 1.20.0, which should be released soon. Please check #8291, where you find a proposal on how to do that which also includes your requirements, and give feedback. We'll implement that on master before branching off 1.20.0, so you'll get that with the latest release.

I'm going to close this ticket because there is no point porting this feature in a future 1.19.3, since we are going to release 1.20.0 anytime soon. Please continue the discussion in #8291. Thanks!

**Comment by casella** [comment]
> @casella I think the main issue from the usability point of view was not actually that one has to **select** the MSL via the systems library setting after the first startup (they are actually installed) but that one has to do this every single time unless one goes into the Tools-> Options and ad there the MSL so it gets loaded by default in the future. That feels unnecessarily complicated, especially for new beginners.

I agree. At this point I believe the best solution to this specific issue is to load MSL 4.0.0 by default, _which is part of my proposed solution._ As I wrote, in the past this would have been a bad idea, because if you then loaded an MSL 3.2.3-compatible package, OMEdit won't unload MSL 4.0.0, and nothing worked. But now this is solved: you get a dialog telling you about the incompatibility, and with the click of a button everything is sorted out, so there's really no inconvenience.

Is that OK for you?

> Having all the "base" libraries like Complex, ModelicaServices, etc next to Modelica without any indication of that they are just dependencies and should not be loaded directly is also not helping. 

That is true, though in principle you may use Complex without MSL. In fact, I am actually doing that in some research projects. Anyway, we took the decision to handle MSL as any other system library using the package manager, because this simplifies things a lot. I think that as soon as we include cached versions of MSL and dependencies without the need of installing them explicitly via the package manager, and then load MSL 4.0.0 by default, all this usability concerns are solved, because everything works out of the box also for beginners.

> Neither is being able to select different versions from the already loaded Modelica one. So as you correctly pointed out, it is kind of a mess and very confusing for new people.

The current situation is messy, we know that. What I was asking is what you think of my proposal (that includes loading MSL 4.0.0, and thus the dependencies, by default) addresses your concerns. Is that the case?

**Comment by dietmarw** [comment]
@casella Yes I'm all for loading MSL 4.0.0 by default when one opens OpenModelica. Especially since the reloading of 3.2.3 is taken care of if needed.
